package.json

```diff
@@ -2,7 +2,7 @@
   "private": true,
   "name": "@okta/okta-auth-js",
   "description": "The Okta Auth SDK",
-  "version": "7.11.1",
+  "version": "7.12.0-rc1",
```

Comment:
releasing rc version of change for testing

Reply:
just make sure bacon is fine with this version format, it's totally valid, just not sure if bacon has issue with it